ci/run_test_main_node.sh

```diff
@@ -27,4 +27,5 @@ export CMOCKA_MESSAGE_OUTPUT=xml
 export CMOCKA_XML_FILE="$DAOS_BASE"/test_results/%g.xml
 cd "$DAOS_BASE"
 IS_CI=true OLD_CI=false utils/run_test.sh
+IS_CI=true OLD_CI=false RUN_TEST_VALGRIND=memcheck utils/run_test.sh
```

Comment:
I would do this as separate thing in run_test stage so valgrind and non-valgrind can run in parallel